main/main.zkasm

```diff
@@ -94,7 +94,7 @@ txLoopRLP:
 
         $ => A          :MLOAD(batchL2DataLength)
         $ => C          :MLOAD(batchL2DataParsed)
-        C - A           :JMPN(loadTx_rlp)
+        C - A           :JMPN(loadTx_rlp) ; TODO CHECK JUMPN only checks 64 bits
```

Comment:
I would remove the comments because the same reason as `load-tx-rlp-utils.zkasm`